src/main/java/kata/makers/ShowsDatabase.java

```diff
@@ -0,0 +1,38 @@
+package kata.makers;
+
+import java.util.ArrayList;
+import java.util.List;
+import kata.makers.exception.RequiredFieldsEmptyException;
+import lombok.Getter;
+
+@Getter
+public class ShowsDatabase {
+
+  private final List<Show> shows = new ArrayList<>();
+
+  public void add(Show show) throws RequiredFieldsEmptyException {
+
+    try {
+      if (!show.getTitle().isBlank() && !show.getYearReleased().isBlank()) {
```

Comment:
This void method would quietly fail if this happens. Is this something you want to implement later and didn't get to yet, or a missed path?